### PR TITLE
Update cleanup_data method

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Fix [OSDEV-660](https://opensupplyhub.atlassian.net/browse/OSDEV-660)

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
This looks cleaner & understandable.

**Comment by VadimKovalenkoSNF** [comment]
> But what happend with tests that touch this area?

Test for **csv** upload files are broken now, because this fix introduced regression. Initially, fix was tended to adjust **xls** file parsing, but it's not appropriate for **csv**, because `cleanup_data()` method is used for both **csv** and **xls** ( which is wrong and should be refactored in Contricleaner re-architecture PR ).

File: src/django/api/helpers/helpers.py [hunk 1]
```diff
@@ -132,13 +132,11 @@ def replace_invalid_data(value, invalid_keywords):
 
 
 def cleanup_data(value):
-    dup_pattern = ',' + '{2,}'
```
Comment (VadimKovalenkoSNF) [inline src/django/api/helpers/helpers.py:135]: This function will be removed as it's not a part of Contricleaner.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — COMMENTED** [review]
LGTM
@VadimKovalenkoSNF, I haven't approved it yet because Mariel hasn't responded to your question under the Jira ticket. Please notify me when the PR is ready for final review.

**Comment by VadimKovalenkoSNF** [comment]
> Please notify me when the PR is ready for final review.

Will do

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**